doc: generate v1.12.0 (#1478)
mongodb/terraform-provider-mongodbatlas: CHANGELOG.md

# Changelog

## [v1.12.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.12.0) (September 19, 2023)

[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.11.1...v1.12.0)

**Enhancements**

> - Support new tags attribute in cluster, advanced\_cluster, and serverless\_instance [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer: Related to this PR, would include an additional **Deprecations and Removals** section and mention the deprecation of `labels` in **cluster** and **advanced_cluster**. (My bad for not creating a separate PR/ticket for the deprecation part)

Author: done in a new commit

> - Support new tags attribute in cluster, advanced\_cluster, and serverless\_instance [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: revising to start with present tense verb + add formatting
```suggestion
- Adds support for the `tags` attribute in `cluster`, `advanced\_cluster`, and `serverless\_instance` [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Support new tags attribute in cluster, advanced\_cluster, and serverless\_instance [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: question: is escaping the underscore necessary? or is this something that must be done because of how the tooling generates the changelog?

> - Sdd database\_user username parameter OIDC footnote in docs [\#1458](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1458) ([Zuhairahmed](https://github.com/Zuhairahmed))

Reviewer 3: s/Sdd/Add/e

Author: thanks

> - Sdd database\_user username parameter OIDC footnote in docs [\#1458](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1458) ([Zuhairahmed](https://github.com/Zuhairahmed))

Reviewer 2: ```suggestion
- Adds `database\_user` username parameter OIDC footnote in docs [\#1458](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1458) ([Zuhairahmed](https://github.com/Zuhairahmed))
```

Author: thx, changed

**Bug Fixes**

> - Populate total\_count in mongodbatlas\_alert\_configurations data source  [\#1476](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1476) ([lantoli](https://github.com/lantoli))

Reviewer 2: present tense verb + formatting
```suggestion
- Populates `total\_count` in `mongodbatlas\_alert\_configurations` data source  [\#1476](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1476) ([lantoli](https://github.com/lantoli))
```

> - Improved error handling for cloud\_backup\_schedule resource. [\#1474](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1474) ([marcosuma](https://github.com/marcosuma))

Reviewer 2: ```suggestion
- Improves error handling for `cloud\_backup\_schedule` resource. [\#1474](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1474) ([marcosuma](https://github.com/marcosuma))
```

> - Handle incorrect ids when importing alert\_configuration or project\_ip\_access\_list resources [\#1472](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1472) ([lantoli](https://github.com/lantoli))

Reviewer 2: ```suggestion
- Handles incorrect ids when importing `alert\_configuration` or `project\_ip\_access\_list` resources [\#1472](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1472) ([lantoli](https://github.com/lantoli))
```

> - Fix when removing collectionName from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))

Reviewer 2: I think we can provide a bit more detail in the description for this item. Feel free to revise if it doesn't capture the issue appropriately
```suggestion
- Fixes issue where removing `collectionName` from a user role doesn't work [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))
```

> - Changelog Spelling Fixes  [\#1457](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1457) ([Zuhairahmed](https://github.com/Zuhairahmed))

Reviewer 2: start with a present tense verb
```suggestion
- Fixes typos in Changelog  [\#1457](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1457) ([Zuhairahmed](https://github.com/Zuhairahmed))
```

**Closed Issues**

- Alert notification interval\_min not working [\#1464](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1464)
- Changing DB user role from readwrite@DB.Col1 to readwrite@DB doesn't work [\#1462](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1462)
- Unable to deploy a DISK\_PARTITION\_UTILIZATION\_DATA AlertConfiguration [\#1410](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1410)
- Error: The terraform-provider-mongodbatlas\_v1.11.0 plugin crashed [\#1396](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1396)
- Trigger resource does not force replacement when app id changes [\#1310](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1310)

**Internal Improvements**

> - Bump goreleaser/goreleaser-action from 4 to 5 [\#1470](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1470) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps goreleaser/goreleaser-action from 4 to 5 [\#1470](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1470) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Bump github.com/aws/aws-sdk-go from 1.45.4 to 1.45.8 [\#1469](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1469) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps github.com/aws/aws-sdk-go from 1.45.4 to 1.45.8 [\#1469](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1469) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Merge feature branch into master [\#1460](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1460) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: question: what was this a feature branch for? If it's an enhancement, I think we can move it to the enhancements section and provide a description of what this feature branch was for

> - Bump github.com/aws/aws-sdk-go from 1.45.2 to 1.45.4 [\#1459](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1459) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps github.com/aws/aws-sdk-go from 1.45.2 to 1.45.4 [\#1459](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1459) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Bump github.com/hashicorp/terraform-plugin-testing from 1.4.0 to 1.5.1 [\#1455](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1455) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps github.com/hashicorp/terraform-plugin-testing from 1.4.0 to 1.5.1 [\#1455](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1455) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Bump github.com/zclconf/go-cty from 1.13.3 to 1.14.0 [\#1454](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1454) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps github.com/zclconf/go-cty from 1.13.3 to 1.14.0 [\#1454](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1454) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Bump github.com/hashicorp/hcl/v2 from 2.17.0 to 2.18.0 [\#1453](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1453) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps github.com/hashicorp/hcl/v2 from 2.17.0 to 2.18.0 [\#1453](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1453) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Bump actions/checkout from 3 to 4 [\#1452](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1452) ([dependabot[bot]](https://github.com/apps/dependabot))

Reviewer 2: ```suggestion
- Bumps actions/checkout from 3 to 4 [\#1452](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1452) ([dependabot[bot]](https://github.com/apps/dependabot))
```

> - Fix docs and example fix for project\_api\_key resource after removing role\_names deprecated field [\#1441](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1441) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Fixes docs and example for `project\_api\_key resource` after removing `role\_names` deprecated field [\#1441](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1441) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Support for new mongodbatlas\_atlas\_user and mongodbatlas\_atlas\_users data sources [\#1432](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1432) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer: We can move this into the enhancements section

Author: done in a new commit

> - Add breaking changes strategy for Terraform [\#1431](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1431) ([wtrocki](https://github.com/wtrocki))

Reviewer 2: ```suggestion
- Adds breaking changes strategy for Terraform [\#1431](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1431) ([wtrocki](https://github.com/wtrocki))
```

> - Extract Configure and Metadata framework functions into single implementation [\#1424](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1424) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Extracts Configure and Metadata framework functions into single implementation [\#1424](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1424) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Fix INTMDB-1017 - Updated alert configuration schema with required params [\#1421](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1421) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: ```suggestion
- Fixes INTMDB-1017 - Updated alert configuration schema with required params [\#1421](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1421) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

> - Remove 1.12.0 deprecated fields. [\#1418](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1418) ([marcosuma](https://github.com/marcosuma))

Reviewer: Include in **Deprecations and Removals**

Author: done in a new commit

Reviewer: Not sure if we want to provide all the details here (attributes of each resource/data source) as described in https://jira.mongodb.org/browse/INTMDB-1015

Author: ok, i can add the list here

> - IP Access List doc updates for Terraform Resources, Data Sources [\#1414](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1414) ([zach-carr](https://github.com/zach-carr))

Reviewer 2: ```suggestion
- Updates IP Access List docs for Terraform Resources, Data Sources [\#1414](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1414) ([zach-carr](https://github.com/zach-carr))
```

> - Avoid diff in state after import for undefined optional attribute in alert config notification [\#1412](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1412) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Avoids diff in state after import for undefined optional attribute in alert config notification [\#1412](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1412) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Migrate Resource: mongodbatlas\_project\_ip\_access\_list to Terraform Plugin Framework [\#1411](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1411) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_project\_ip\_access\_list` resource to Terraform Plugin Framework [\#1411](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1411) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

> - Remove validation of empty public\_key and private\_key attributes in provider config to avoid breaking change [\#1402](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1402) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Removes validation of empty `public\_key and private\_key` attributes in provider config to avoid breaking change [\#1402](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1402) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Migrate DataSource mongodbatlas\_alert\_configuration to Terraform Plugin Framework [\#1397](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1397) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_alert\_configuration` data source to Terraform Plugin Framework [\#1397](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1397) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Migrate DataSource: mongodbatlas\_project\_ip\_access\_list to Terraform Plugin Framework [\#1395](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1395) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_project\_ip\_access\_list` data source to Terraform Plugin Framework [\#1395](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1395) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

> - Migrate Resource: mongodbatlas\_database\_user to Terraform Plugin Framework [\#1388](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1388) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_database\_user` resource to Terraform Plugin Framework [\#1388](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1388) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

> - Update project documentation to remove api\_keys references [\#1386](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1386) ([maastha](https://github.com/maastha))

Reviewer 2: ```suggestion
- Updates project documentation to remove `api\_keys` references [\#1386](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1386) ([maastha](https://github.com/maastha))
```

> - Migrate alert\_configuration resource and removing deprecated fields [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer: This PR includes removal of deprecated fields to include in **Deprecations and Removals**

Author: done in a new commit

> - Migrate alert\_configuration resource and removing deprecated fields [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Migrates `alert\_configuration` resource and removes deprecated fields [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Merge master into CLOUDP-18958 [\#1384](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1384) ([andreaangiolillo](https://github.com/andreaangiolillo))

Reviewer 2: question: can we provide a clearer description of what this commit includes?

Author: thx, added a better description

> - Migrate mongodbatlas\_encryption\_at\_rest to Terraform Plugin Framework [\#1383](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383) ([maastha](https://github.com/maastha))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_encryption\_at\_rest` resource to Terraform Plugin Framework [\#1383](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383) ([maastha](https://github.com/maastha))
```

> - New framework provider, main and acceptance tests to use mux server with existing sdk v2 provider [\#1366](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1366) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 2: ```suggestion
- Adds new framework provider, main and acceptance tests to use mux server with existing sdk v2 provider [\#1366](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1366) ([AgustinBettati](https://github.com/AgustinBettati))
```

> - Migrate Resource: mongodbatlas\_project to Terraform Plugin Framework and remove api\_keys attribute [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))

Reviewer: This PR includes removal of deprecated fields to include in Deprecations and Removals

Author: done in a new commit

> - Migrate Resource: mongodbatlas\_project to Terraform Plugin Framework and remove api\_keys attribute [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))

Reviewer 2: ```suggestion
- Migrates `mongodbatlas\_project` resource to Terraform Plugin Framework and remove api\_keys attribute [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))
```

- Improved error handling for cloud\_backup\_schedule resource. [\#1474](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1474) ([marcosuma](https://github.com/marcosuma))
- Handle incorrect ids when importing alert\_configuration or project\_ip\_access\_list resources [\#1472](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1472) ([lantoli](https://github.com/lantoli))
- Fix when removing collectionName from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))
- Changelog Spelling Fixes  [\#1457](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1457) ([Zuhairahmed](https://github.com/Zuhairahmed))

**Deprecations and Removals**

- Deprecation of labels attributes in cluster and advance_cluster resources.
- Remove 1.12.0 deprecated fields. [\#1418](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1418) ([marcosuma](https://github.com/marcosuma))
- Remove deprecated fields in alert\_configuration resource [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))

> - Remove api\_keys attribute from mongodbatlas\_project [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))

Reviewer 4: Another removal was in encryption_at_rest (removed deprecated fields `aws_kms`, `azure_key_vault`, `google_cloud_kms`)
My bad, should have been a separate PR.

PR https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383/files#diff-72229c36539e51cfa6fbd6c54b91a9b3ab72a524afce9428734c0c222c5f4528

Author: thx, added in new commit

# Changelog

> ## [v1.12.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.12.0) (September 19, 2023)

Reviewer 5: ```suggestion
## [v1.12.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.12.0) (2023-09-20)
```

[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.11.1...v1.12.0)

**Enhancements**

> - Adds support for the `tags` attribute in `mongodbatlas_cluster`, `mongodbatlas_advanced_cluster`, and `mongodbatlas_serverless_instance`. See [Atlas Resource Tags](https://www.mongodb.com/docs/atlas/tags/) to learn more. [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))

Reviewer 5: ```suggestion
- Support for `tags` attribute in `mongodbatlas_cluster`, `mongodbatlas_advanced_cluster`, and `mongodbatlas_serverless_instance`. See [Atlas Resource Tags](https://www.mongodb.com/docs/atlas/tags/) to learn more. [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))
```

- Support for new `mongodbatlas_atlas_user` and `mongodbatlas_atlas_users` data sources [\#1432](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1432) ([AgustinBettati](https://github.com/AgustinBettati))

**Bug Fixes**

- Populates `total_count` in `mongodbatlas_alert_configurations` data source  [\#1476](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1476) ([lantoli](https://github.com/lantoli))
- Improves error handling for `cloud_backup_schedule` resource. [\#1474](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1474) ([marcosuma](https://github.com/marcosuma))
- Handles incorrect ids when importing `alert_configuration` or `project_ip_access_list` resources [\#1472](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1472) ([lantoli](https://github.com/lantoli))

> - Fix when removing collectionName from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))

Reviewer 5: ```suggestion
- Fix when removing `collectionName` from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))
```

- Changelog Spelling Fixes  [\#1457](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1457) ([Zuhairahmed](https://github.com/Zuhairahmed))
- Adds `mongodbatlas_database_user` username parameter OIDC footnote in docs [\#1458](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1458) ([Zuhairahmed](https://github.com/Zuhairahmed))

**Deprecations and Removals**

- Deprecation of `labels` attribute in `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster` resources.
- Remove deprecated fields in `mongodbatlas_alert_configuration` resource [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))

> - Remove api\_keys attribute from mongodbatlas\_project [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))

Reviewer 5: ```suggestion
- Remove `api_keys` attribute from `mongodbatlas_project` [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))
```

> - Remove api\_keys attribute from mongodbatlas\_project [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))

Reviewer 5: ```suggestion
- Removal of api\_keys attribute from mongodbatlas\_project [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))
```

> - Remove attributes in encryption_at_rest resource: aws_kms, azure_key_vault, google_cloud_kms [\#1383](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383) ([maastha](https://github.com/maastha))

Reviewer 5: ```suggestion
- Removal of attributes in `mongodbatlas_encryption_at_rest` resource: aws_kms, azure_key_vault, google_cloud_kms [\#1383](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383) ([maastha](https://github.com/maastha))
```

> - Remove 1.12.0 deprecated fields. [\#1418](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1418) ([marcosuma](https://github.com/marcosuma))

Reviewer 5: ```suggestion
- Removal of MongoDB Atlas Terraform Provider v1.12.0 deprecated fields. [\#1418](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1418) ([marcosuma](https://github.com/marcosuma))
```
